### PR TITLE
More informative secrets.yml.sample

### PR DESCRIPTION
**Pull request by stanleypliu**
Updated `secrets.yml.sample` with comments. Also see LastPass

### CODE REVIEW

File: config/secrets.yml.sample [hunk 1]
```diff
@@ -1,21 +1,29 @@
-slack_token: XXXXXXXXXX
+# Details for the below keys are in LastPass under Shared Folders > Labs (secure note)
+# Also see the note for more information on what each key does
```
Comment (defaye) [inline config/secrets.yml.sample:2]: I can't seem to see this shared folder

**Review by defaye — CHANGES_REQUESTED** [review]
I'm having some problems seeing the file in LastPass. We might need to go over permissions, but also see comment on my suggested shared folder

File: config/secrets.yml.sample (hunk 1 above)
Comment (stanleypliu) [reply config/secrets.yml.sample:2]: Yeah unfortunately it seems to have disappeared. Will create it again

Comment (defaye) [reply config/secrets.yml.sample:2]: As per conversation in Slack, I've added you as an administrator (including others that were missing) to this folder, which as you discovered(!) you needed to then re-login to see the folder in your vault

File: config/secrets.yml.sample [hunk 2]
```diff
@@ -1,21 +1,29 @@
-slack_token: XXXXXXXXXX
+# Details for the below keys are in LastPass under Shared Folders > Labs (secure note)
```
Comment (defaye) [inline config/secrets.yml.sample:1]: Can we change this to Development Environments instead now?

**Review by defaye — APPROVED** [review]

**Pull request merged**